languages/csharp/exercises/concept/numbers-floating-point/.docs/introduction.md

```diff
@@ -11,3 +11,5 @@ C# has three floating point types:
 - `decimal`: 16 bytes (28-29 digits precision). Normally used when working with monetary data, as its precision leads to less rounding errors. Written as `2.45m`.
 
 As can be seen, each type can store a different number of digits. This means that trying to store PI in a `float` will only store the first 6 to 9 digits (with the last digit being rounded).
+
+There are several ways to write loops in C#, the most common ones being `while` and `for` loops.
```

Comment:
```suggestion
In this exercise you may also want to use a loop. There are several ways to write loops in C#, the most common ones being `while` and `for` loops.
```

Suggest making `while` and `for` hyperlinks to the docs?

Reply:
I'm still not sure there. My thoughts were: I'll introduce the keywords and students can lookup and explore them themselves. Then if they need help, the hints will point to an actual documentation page.

@SleeplessByte Suggested that we might link to v3 reference documentation.

Reply:
Arguing against myself, if we look at [this PR](https://github.com/exercism/v3/pull/116/files), it says:

> Each mention should focus on how that Concept is applied in the given language, with links to learn more on the topic. If there is nothing special about it, a language-agnostic link is helpful.

That would argue for including the link.

Reply:
If I look back at my Exercism experiences as a student, I found that the best experience was when people pointed me in the right direction, but I still had to do the searching and discovering myself. Hence my reluctance to start with links to resources in the introduction.